### ISSUE TITLE
global incorrectly redefined

### ISSUE THREAD

**Issue by crowmagnumb** [NativeScript/NativeScript#8677]
**Environment**

```
$ tnsinfo
✔ Getting NativeScript components versions information...
✔ Component nativescript has 6.7.6 version and is up to date.
✔ Component @nativescript/core has 6.5.8 version and is up to date.
⚠ Update available for component tns-android. Your current version is 6.5.0 and the latest available version is 6.5.1.
⚠ Update available for component tns-ios. Your current version is 6.5.0 and the latest available version is 6.5.1.
```

```
"@nativescript/core": "^6.5.8"
"nativescript-dev-webpack": "~1.5.1"
"@ngtools/webpack": "^9.1.9"
"typescript": "~3.8.3"
```

I'm including the following since it's the ```@types/node``` at version **14,0,14** that causes this. I used to have version **14.0.1** because I had ```@types/webpack-sources@0.1.7``` (now 0.1.8).
```
$ npm list @types/node
@animalus/ng-core@1.3.0 /home/ken/dev/animalus/ng-core
├─┬ @angular-devkit/build-angular@0.901.9
│ ├─┬ license-webpack-plugin@2.1.4
│ │ └─┬ @types/webpack-sources@0.1.8
│ │   └── @types/node@14.0.14 
│ └─┬ webpack-dev-server@3.11.0
│   └─┬ del@4.1.1
│     └─┬ @types/glob@7.1.2
│       └── @types/node@14.0.14  deduped
└─┬ ng-packagr@9.1.5
  └─┬ @rollup/plugin-node-resolve@7.1.3
    └─┬ @types/resolve@0.0.8
      └── @types/node@14.0.14  deduped
```

**Describe the bug**

With everything right up to the latest greatest, I get the following error when I run ```tns run android``` ...

```
ERROR in node_modules/@types/node/ts3.5/globals.global.d.ts:1:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'global' must be of type 'Global', but here has type 'Global & typeof globalThis'.

1 declare var global: NodeJS.Global & typeof globalThis;
              ~~~~~~

  node_modules/@nativescript/core/module.d.ts:2:13
    2 declare var global: NodeJS.Global;
                  ~~~~~~
    'global' was also declared here.
```

**Workaround**

If I manually delete the second line of ```node_modules/@nativescript/core/module.d.ts``` then it works. I could of course make it equal to the same as that in ```node_modules/@types/node/ts3.5/globals.global.d.ts``` but deleting is easier and works.

**Comment by Ross-Olivares** [comment]
Hi @crowmagnumb 
Thank you for reporting this issue.
Before we try to replicate this on our end, can you confirm if you are still facing this issue?

**Comment by crowmagnumb** [comment]
Yes, I have sent instructions to @NathanWalker about this. Here is a [repo](https://github.com/animalus/shared-lib-example) that demonstrates. If you clone that, checkout the "alias_issue" branch, and then follow the instructions for installing and running in the README you should see the issue. NOTE: The package.json file is different in that branch so be sure to do a clean npm install after checking out that branch. Also I did not do an audit fix after npm install if you want to make sure you have the exact same setup as me.

**Comment by crowmagnumb** [comment]
By the way I only get this with `tns run`. Building my code with `ng build` works fine.

**Comment by kevensaldana** [comment]
I fixed it with npm i @types/node@14.0.1 --save for the moment.
